### ISSUE TITLE
Cross Origin Headers should be added

### ISSUE THREAD

**Issue by eranchetz** [logstash-plugins/logstash-input-http#29]
Logstash http input should be able to respond with CORS headers : 
Access-Control-Allow-Origin: * 
So you can send logs via ajax (XHR) http request

**Comment by arabold** [comment]
Oh, I like this idea. This could allow us to use Logstash and ElasticSearch as a client-side tracking platform for web sites.

**Comment by mkuzmin** [comment]
Yes, I'm also setting up ELK to collect JavaScript reports from browsers, and need this feature too.

**Comment by jordansissel** [comment]
We could allow custom headers to be set by the user in a special setting, thoughts?

```
input {
  http {
    # proposal
    headers => {
      "Access-Control-Allow-Origin" => "example.com"
      "Vary" => "Origin"
    }
    ...
  }
}
```

**Comment by mkuzmin** [comment]
I like the idea with customizable headers.
For my current installation I've hadcoded it this way: https://github.com/mkuzmin/logstash-input-http/commit/084ec987d9029bfe9cd6018a33983836dea94b17

**Comment by babadofar** [comment]
Customizable headers would be perfect!

**Comment by totev** [comment]
+1

**Comment by suyograo** [comment]
Fixed in #42. Available in 2.1.1
